docs-website/pages/plugins-manifest.md

```diff
@@ -0,0 +1,206 @@
+
+#### `compatibleVersion`
+
+Defines the minimum version of Sketch required to run the plugin. This string must be provided using [semantic versioning](semantic versioning).
+
+```json
+"compatibleVersion": "52.1"
+```
+
+#### `commands`
+
+An array of dictionaries defining all commands provided by the plugin.
+
+```json
+"commands": [
+  {
+    "name": "All",
+    "identifier": "all",
+    "shortcut": "ctrl shift a",
+    "script": "shared.js",
+    "handler": "selectAll"
+  }
+]
+```
+
+##### Example handler definitions
+
+```js
+// default handler
+export default function(context) {
+  console.log(context.selection)
+}
+
+// explicitly defined handler
+export function selectAll(context) {
+  console.log(context.document)
+}
+```
+
+| Member       | Description                                                                                                                                                                                                                                           |
+| ------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `identifier` | Defines a unique identifier of the command within the plugin bundle.                                                                                                                                                                                  |
+| `name`       | Provides the name of the command which is used within the _Plugins_ menu.                                                                                                                                                                             |
+| `shortcut`   | Provides a default keyboard shortcut for the command, e.g. `ctrl shift t`.                                                                                                                                                                            |
+| `script`     | Specifies the relative path within the plugin bundle's `Sketch` folder to the script implementing the command.                                                                                                                                        |
+| `handler`    | Specifies the name of function to be called with the command. The function must accept a single `context` parameter, containing information such as the current document and selection. If unspecified the command is expected to be `export default` |
+
+#### `description`
+
+Provides a general description of what the plugin does.
+
+#### `disableCocoaScriptPreprocessor`
+
+Defines if the CocoaScript preprocessor should be used for the plugin's scripts. Set this to `true` to use ES6 syntax and build systems like [skpm](https://skpm.io).
+
+When disabled, `@import` and bracket syntax, i.e. `[obj hello: world]` are no longer supported.
```

Comment:
this line is a little confusing because of the parameter name `disableCocoaScriptPreprocessor`

Maybe, 

"When set to `true`, `@import` and bracket syntax, i.e. `[obj hello: world]` are supported"